std/algorithm/sorting.d

```diff
@@ -145,20 +145,20 @@ Checks whether a forward range is sorted according to the comparison
 operation $(D less). Performs $(BIGOH r.length) evaluations of $(D
 less).
 
-Unlike $(LREF isSorted), $(LREF isStrictlyMonotonic) does not allow for equal values,
+Unlike isSorted, isStrictlyMonotonic does not allow for equal values,
```

Comment:
> What's auto-highlighting

Ddoc recognizes and highlights certain words in prose (keywords, the symbol being documented, parameters). I call that auto-highlighting, as opposed to doing it manually with `$(D ...)` or backticks. Auto-highlighting is a common source of broken links, because it happens in URLs too (as happened here).

> and why doesn't ddox have it as well?
> Didn't ddox automatically detect symbols in the past?

DDOX does some auto-linking of symbols, but I'm not sure how it detects them. I don't think it ever searched for the symbol that's being documented or its parameters to highlight them, like Ddoc does.

DDOX not doing auto-highlighting is a problem which I didn't have on my radar. There should be consistency across Ddoc and DDOX. As far as I see, we can either:

* implement auto-highlighting in DDOX, or
* disable auto-highlighting in our Ddoc macros and highlight everything manually.

I'm not up for either of those challenges right now, so I've just added backticks to this PR. So that at least the symbols here are in monospaced font (and auto-linked) on the DDOX pages, too.